experiments/tbats/README.md

```diff
@@ -1,70 +1,89 @@
 # TBATS: A brand-new implementation for the Python ecosystem
 
-## TL;DR 
-For next-day forecasting of electricity load, StatsForecast's new implementation of TBATS outperforms current implementations of this model in both accuracy and speed.
+## TL;DR
```

Comment:
I think results are ok for the M3 dataset since we're close to R's. Yearly performs worse than the seasonal naive, but R is no better. The results for the M4 dataset suggests we should keep checking the code, in particular for hourly. 

I removed the original dataset (electricity) because results got worse after fixing a couple of bugs in the previous version.